VirtoCommerce.PricingModule.Web/Scripts/blades/prices-list.js

```diff
@@ -181,7 +181,7 @@
         },
         isUniqueQtyForPricelist: function (data) {
             // Disable unique quantity test when time filtering is used.
-            return _.filter(allPrices, function (price) { return price.pricelistId == data.pricelistId && price.minQuantity == data.minQuantity && !price.startDate && !price.EndDate && !data.startDate && !data.endDate; }).length <= 1;
+            return Math.round(data.minQuantity) > 0 && _.filter(allPrices, function (price) { return price.pricelistId == data.pricelistId && price.minQuantity == data.minQuantity && !price.startDate && !price.EndDate && !data.startDate && !data.endDate; }).length <= 1;
```

Comment:
Could you please leave only changes related to the task in this file and revert formatting changes in places that were not touched? Otherwise when you are looking at the commit it is harder to find what is the real change for fixing problem.

Reply:
it`s done.